### ISSUE TITLE
Error occured when finalizing GeneratorDataset iterator

### ISSUE THREAD

**Issue by SIlvaMFPedro** [AntonMu/TrainYourOwnYOLO#195]
Good afternoon.
I seem to be having a problem generated by Tensorflow during the training process of my YOLO detector for foot detection using a custom dataset:
![image](https://user-images.githubusercontent.com/22821533/96140289-ad2c3a80-0ef7-11eb-9ed9-6a21d5078ee7.png)
I am using Tensorflow 2.3.0 and i am training the network with 20 epochs, a batch size of 32 and a learning rate 1e-4.
Have you ever had this issue before? And if so can you help me solve this?
Thank you very much.

Pedro Silva

**Comment by AntonMu** [comment]
#191

**Comment by K1ndWha1e** [comment]
I have same mistake, but on 88'th epoch and my yolo can detect my classes on images...
